HoloRepositoryDemoApplication/Assets/Scripts/LoginAuthentication.cs

```diff
@@ -8,11 +8,11 @@
 public class LoginAuthentication : MonoBehaviour
```

Comment:
I noticed that it is a bit cumbersome to always enter the number when you are developing. Maybe you could have an ooptional flag / boolean / checkbox in the inspector like "Skip in development mode" and then if it is true and you are in Unity game mode, skip it? Maybe [this](https://docs.unity3d.com/ScriptReference/Application-isPlaying.html) or [this](https://docs.unity3d.com/ScriptReference/EditorApplication-isPlaying.html) could be used.

Reply:
done